setup.py

```diff
@@ -11,6 +11,13 @@
     from pip.req import parse_requirements
 
 
+def _resolve_requirement(req) -> str:
+    if req.__class__.__name__ == "ParsedRequirement":
+        return req.requirement
+    else:
+        return f"{x.name}{x.specifier}"
```

Comment:
where is this x from?

Reply:
Copy/paste from the previous implementation. Fixed now. ty